### PR TITLE
Update job-config.json for 3.9.1

### PR DESCRIPTION
**Pull request by SDawley**
I think just updating the CSV here is all thats needed?

### CODE REVIEW

File: dependencies/job-config.json [hunk 1]
```diff
@@ -517,29 +517,29 @@
...
       },
       "3.10": {
         "CSV_VERSION": "3.10.0",
-        "CSV_VERSION_PREV": "3.8.0"
+        "CSV_VERSION_PREV": "3.9.0"
       },
       "3.x": {
-        "CSV_VERSION": "3.10.0",
-        "CSV_VERSION_PREV": "3.8.0"
+        "CSV_VERSION": "3.11.0",
```
Comment (nickboldt) [inline dependencies/job-config.json:528]: not needed on the 3.9 branch... only need stuff that the 3.9 job cares about. but also won't hurt if you apply this.

**Review by nickboldt — APPROVED** [review]

**Comment by SDawley** [comment]
If not needed then I'll close.

**Comment by nickboldt** [comment]
> If not needed then I'll close.

Sorry for not being clear -- I meant adding info about 3.11 to the 3.9 branch was not needed.

Updating 3.9.1 replaces 3.9.0... that IS required.

**Comment by SDawley** [comment]
in that case, merge.

**Pull request merged**